src/CodeGeneration.Roslyn.Tasks/CodeGeneration.Roslyn.Tasks.csproj

```diff
@@ -27,4 +29,27 @@
     <PackageReference Include="Nerdbank.MSBuildExtension" Version="0.1.17-beta" PrivateAssets="all" />
   </ItemGroup>
 
+  <Target Name="SetPackageVersionInProps" DependsOnTargets="GetBuildVersion">
+    <PropertyGroup>
+      <PropsFileDestinationDirectory>$(IntermediateOutputPath)build\</PropsFileDestinationDirectory>
+      <PropsFilePath>$(PropsFileDestinationDirectory)\$(PropsFileName)</PropsFilePath>
+      <_ProjectNamespaces>
+        <Namespace Prefix="dn" Uri="http://schemas.microsoft.com/developer/msbuild/2003" />
+      </_ProjectNamespaces>
+    </PropertyGroup>
+    <MakeDir Directories="$(PropsFileDestinationDirectory)" />
+    <Copy SourceFiles="build\$(PropsFileName)" DestinationFiles="$(PropsFilePath)" />
+    <XmlPoke XmlInputPath="$(PropsFilePath)" Value="$(PackageVersion)" Query="/dn:Project/dn:ItemGroup/dn:DotNetCliToolReference/@Version" Namespaces="$(_ProjectNamespaces)" />
+    <ItemGroup>
+      <None Remove="build\$(PropsFileName)" />
+      <None Include="build\$(PropsFileName)">
```

Comment:
Why remove and include when you can Update?

Reply:
Updating doesn't work for me. When I do just Update, all items in group have Pack set to false, instead of only the one.

Also it's very restricted as per docs: https://docs.microsoft.com/pl-pl/visualstudio/msbuild/item-element-msbuild#attributes

Reply:
Hmmm.... it seems like it ought to have worked here. All the "restrictions" are satisfied in this case as far as I can tell. Oh well, thanks for trying.